### PR TITLE
Fb 1.3.4 snapshot

### PR DESCRIPTION
**Pull request by labkey-susanh**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -8,6 +8,15 @@ but also make certain assumptions that you may not want to impose on your module
 [LabKey documentation](https://www.labkey.org/Documentation/wiki-page.view?name=gradleModules) for more information.
 
 ## Release Notes
+### version 1.3.4
+*Release*: ???
+(Earliest compatible LabKey version: 18.2)
```
Comment (labkey-tchad) [inline README.md]: Should this be bumped to 18.3 or will we backport today's module.xml fix from trunk?

Comment (labkey-tchad) [reply README.md]: If so, we should remove the test dependency on `com.googlecode.sardine` (TestRunner.groovy:line 104)

**Review by labkey-tchad — CHANGES_REQUESTED** [review]
Looks good. Just one question about the supported labkey version

File: README.md (hunk 1 above)
Comment (labkey-susanh) [reply README.md]: Yes, should be bumped to 18.3.  I've made these two changes.  I'll let TC run through a build or two before merging.

**Review by labkey-tchad — APPROVED** [review]

**Pull request merged**